scripts/qa_crowbarsetup.sh

```diff
@@ -2517,6 +2517,9 @@ function custom_configuration
             if [[ $hacloud = 1 ]] ; then
                 proposal_set_value glance default "['deployment']['glance']['elements']['glance-server']" "['cluster:$clusternameservices']"
             fi
+            if iscloudver 7plus && [[ $want_magnum = 1 ]]; then
+                proposal_set_value glance default "['attributes']['glance']['enable_v1]" true
```

Comment:
I wonder if we need to add this check in the barclamp too when magnum and heat are enabled